js/help.js

```diff
@@ -0,0 +1,49 @@
+const helpOverlay = new Image();
+helpOverlay.src = 'images/help-image.png';
+
+const helpBtn = document.querySelector('.help');
+
+let helpImage = false;
```

Comment:
Better to start boolean variables with "is-" or "has-" prefixes.

https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig

Think about functions as verbs, booleans as a kind of binary status, and other types as nouns.